File: Godeps/_workspace/src/github.com/yhat/ops2/src/mps/integration/python/modelenv/hello.py
import os

from yhat import Yhat, YhatModel, preprocess

class HelloWorld(YhatModel):

    VERSION = int(os.environ["MODEL_VERSION"])

    @preprocess(in_type=dict, out_type=dict)
    def execute(self, data):
        return { "version": self.VERSION }


username = os.environ["USERNAME"]
apikey = os.environ["APIKEY"]
endpoint = os.environ["OPS_ENDPOINT"]

print "%s:%s:%s" % (username, apikey, endpoint,)

yh = Yhat(
    username,
    apikey,
    endpoint
)
yh.deploy("modelenvvars", HelloWorld, globals(), sure=True)

File: Godeps/_workspace/src/github.com/yhat/ops2/src/mps/integration/r/echo/hello.r
library(yhatr)

model.transform  <- function(data) {
    data
}
model.predict <- function(data) {
    data
}

username <- Sys.getenv("USERNAME")
print(username)
apikey <- Sys.getenv("APIKEY")
print(apikey)
endpoint <- Sys.getenv("OPS_ENDPOINT")
print(endpoint)

yhat.config  <- c(
    username=username,
    apikey=apikey,
    env=endpoint
)
yhat.deploy("REcho")
